desk/app/groups.hoon

```diff
@@ -1264,17 +1289,19 @@
     ^+  go-core
     =/  user-join  &((~(has in ships) src.bowl) =(1 ~(wyt in ships)))
     =/  am-host  =(p.flag our.bowl)
+    =*  cordon  cordon.group
```

Comment:
Curious to know as a new hooner: what is the purpose of using a deferred expression here vs `=/`?

Reply:
@Fang- any insight here?

Reply:
It's just "alias" vs "new variable" basically. `cordon.group` might be too long for how frequently you want to use it, so you alias it to something shorter (`cordon`). In most cases this is more appropriate than `=/`, because of that semantic difference, and also changing `cordon` here would update the original in `cordon.group` I believe.

Reply:
Wonderful, thanks!